app/build.gradle

```diff
@@ -96,7 +96,8 @@ dependencies {
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
       'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.2.1',
       'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1',
-      'org.mockito:mockito-core:2.7.22'
+      'org.mockito:mockito-core:2.7.22',
+      "com.github.chrisbanes:PhotoView:2.3.0"
```

Comment:
Please use ``'`` instead of ``"`` for consistency.

Reply:
done